packages/lib-panoptes-js/src/resources/collections/commonRequests.js

```diff
@@ -0,0 +1,29 @@
+const panoptes = require('../../panoptes')
+const { endpoint } = require('./helpers')
+const { raiseError } = require('../../utilityFunctions')
+
+function addSubjects (params) {
+  const collectionId = params && params.id
+  if (!collectionId) return raiseError('Collections add subject: collections ID is required.', 'error')
+  const authorization = (params && params.authorization) ? params.authorization : ''
+  const subjects = params && params.subjects
+  if (!subjects || !Array.isArray(subjects)) return raiseError('Collections add subjects: subjects array is required.', 'error')
```

Comment:
Non-blocking comment: Should we also check the ids in the subjects ids array are strings? Or should we just rely on Panoptes to error if that is malformed?

Reply:
I'm inclined to let Panoptes handle errors, rather than couple the client tightly to the API.